contents/handbook/company/team/joe-martin.md

```diff
@@ -0,0 +1,34 @@
+# Bio
+
+[This is my fifth start-up](https://www.linkedin.com/in/joemartinwords/). 
+
+Before I took up Product Marketing I was a Content Marketer, a Copywriter and a Journalist. Before _that_ I was clown, a chainsaw salesman and a cleaner in a morgue. I've had a strange life. 
+
+I'm engaged and live just outside of London. I'm passionate about punk music, videogame preservation and garlic. I can't ride a bike. 
+
+# How I can help you
+- I can help you figure out the right way to communicate something, to anyone, no matter how technical. 
+- I can help you with a broad range of marketing tasks, including sales enablement. 
+- I can help you understand our customers, our competitors or the press. 
+- I can help with literally anything involving writng, editing or proof-reading. 
+- I can help you find new videogames to play.
+
+# How to work with me
+- In meetings I default to listening, rather than speaking. Its rare for me to interupt, so you may need to ask me for my thoughts if you want them.
+- I fidget constantly, often without realising it. If it bothers you then please let me know. 
+- I'll sometimes repeat things back to you ('_It sounds like..._') to make sure I understand. If I'm wrong, I need you to tell me. 
+- I'm truly uncomfortable with public praise or attention. A private thanks is better than a spotlight.
+- I care deeply about my co-workers and trust them by default. I'm always happy to chat, about anything.
+- I'm usually fine with ad-hoc meetings, even on no-meeting days, but weekends are sacred to me. Slack is the best way to reach me. I generally work 9-6, with no break for lunch.
```

Comment:
You might want to add your time zone to 9-6 :)

Reply:
Also, please have lunch!

Reply:
Assumed that would be obvious with the London mention, but good spot. 

(I do have lunch, I just nearly always take it at my desk and work through by choice!)

Reply:
Yeah, good point. 

Good, please look after yourself :)